### PR TITLE
Migrate gometalinter to golangci-lint

### PR DESCRIPTION
**Pull request by fejta**
https://github.com/alecthomas/gometalinter has been replaced by https://github.com/golangci/golangci-lint#comparison-with-gometalinter

### CODE REVIEW

File: verify/go-tools/verify-govet.sh [hunk 1]
```diff
@@ -17,4 +17,4 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-go vet -v $(go list ./... | grep -v /vendor/)
+go vet -v $(go list ./... | grep -v -E '/vendor/|verify$')
```
Comment (BenTheElder) [inline verify/go-tools/verify-govet.sh]: do we need this if we also have golangci-lint running vet?

Comment (fejta) [reply verify/go-tools/verify-govet.sh]: That sounds like sound reasoning... but wouldn't it also apply to gometalinter (which also ran vet/vetshadow)?

Let me investigate and send a follow up.

**Comment by fejta** [comment]
Yeah let me remove it:

```
Verifying /go/src/k8s.io/repo-infra/verify/go-tools/verify-gometalinter.sh
cmd/kazel/generator_test.go:82:4:error: Error call has possible formatting directive %v (vet)

cmd/kazel/generator_test.go:82:4: Error call has possible formatting directive %v
FAILED   /go/src/k8s.io/repo-infra/verify/go-tools/verify-govet.sh	0s
```

**Pull request merged**